### PR TITLE
Documentation update: ethereum

### PR DESCRIPTION
**Pull request by anastasiyaig**
Updated documentation for Ethereum.

The only thing not present is eth subscribe / unsubsribe which are really web sockets methods and most important - are not available for free RPC which we are using in our examples. Please lemme know if I need to add them too, otherwise the methods list is aligned

### CODE REVIEW

File: apis/ethereum/eth_sendRawTransaction.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+```bash
+curl --location 'https://rpc.eth.gateway.fm' \
+--header 'Content-Type: application/json' \
+--data '{
+    "id": 6008149,
+    "jsonrpc": "2.0",
+    "method": "eth_sendRawTransaction",
+    "params": [
+        "0xf8687884f46109038252089447ac4f3a5ea94b648672648e730bfe48ed6e734985e8d4a51000802ca0ee5f7c40c98ce6f0f2aa05ebb65e96ddf402f1152fbb8b76893ddb2b0fd6f1d0a068e60320ad7ab56a5200d55ba97c783d4ad37945eb74d1f5d3100155f059fbae"
```
Comment (Serhy) [inline apis/ethereum/eth_sendRawTransaction.md]: This TX returns {"code":-32000,"message":"INTERNAL_ERROR: rlp parse transaction: invalid chainID, 4 (expected 1)"}} 
Means it created from different network. Not ideal example.

File: apis/ethereum/eth_call.md [hunk 2]
```diff
@@ -19,33 +16,43 @@ This is one of the most commonly used API calls. It is used to read from the blo
...
--H "Content-Type: application/json" \
--d '{"jsonrpc":"2.0","method":"eth_call","params":[{"from": "0x4c88153de66e84c6691fa6bf5b5823530300a942","to": "0x3d14de87b5ade1c61a0b7ca29f7632e6e756b8bf","gas": "0x76c0","gasPrice": "0x9184e72a000","value": "0x0","data": "0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675"}, "latest"],"id":1}'
+curl --location 'https://rpc.eth.gateway.fm' \
+--header 'Content-Type: application/json' \
+--data '{
+    "jsonrpc": "2.0",
+    "method": "eth_call",
+    "params": [
+        {
+            "from": "0x55BA4cB41D0C8F62aB4BB207d0bF9fA42f9FE70d",
```
Comment (Serhy) [inline apis/ethereum/eth_call.md]: Let's replace "from" to 0x00000000219ab540356cBB839Cbe05303d7705Fa value

**Review by Serhy — CHANGES_REQUESTED** [review]

**Review by Serhy — APPROVED** [review]

File: apis/ethereum/eth_sendRawTransaction.md (hunk 1 above)
Comment (anastasiyaig) [reply apis/ethereum/eth_sendRawTransaction.md]: okay i fixed it

File: apis/ethereum/eth_call.md (hunk 2 above)
Comment (anastasiyaig) [reply apis/ethereum/eth_call.md]: done

**Pull request merged**